components/brave_rewards/browser/rewards_service_browsertest.cc

```diff
@@ -308,6 +330,49 @@ class BraveRewardsBrowserTest
       verified.c_str());
   }
 
+  static std::vector<double> GetSiteBannerTipOptions(
+      content::WebContents* site_banner) {
+    auto options = content::EvalJs(
+        site_banner,
+        R"(
```

Comment:
not sure that I know this syntax. Can you please tell me why `R`?

Reply:
prefix(optional) R "delimiter( raw_characters )delimiter"	(6)	(since C++11)